.env.demo

```
@@ -198,4 +198,25 @@ S3_BUCKET_ARN=arn:aws:s3:::xxxxx  // Please provide your AWS bucket arn
+# SENDGRID_API_KEY=SG.xxxxxxx
+
+# AWS SES (Fallback option)
+# AWS_SES_REGION=ap-south-1
+# AWS_SES_ACCESS_KEY=xxxx
+# AWS_SES_SECRET_KEY=xxxx
+
+# SMTP (Fallback option)
+# SMTP_HOST=smtp.gmail.com
+# SMTP_PORT=587
+# SMTP_USER=your-email@gmail.com
+# SMTP_PASS=xxxxxx
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)